goco-transit/src/components/offers.js

```diff
@@ -54,6 +63,16 @@ class Offers extends React.Component {
     this.setState({ deleteOpen: false });
   };
 
+  // Limits seat maximum to pre-defined constant
+  handleSeatPlus = () => {
+    if (this.state.seats < 10) { this.setState({ seats: this.state.seats + 1 }) }
```

Comment:
This is better. I still prefer to split if-clauses onto multiple lines, but at least this in-line one is readable.

Reply:
I split them up if it's more than one statement, but I don't mind either way for one-liners. We can go through towards the end and standardize our syntax if we want.